dvc/objects/db/base.py

```diff
@@ -229,7 +229,7 @@ def list_hashes_traverse(
         keeping all of it in memory.
         """
         num_pages = remote_size / self.fs.LIST_OBJECT_PAGE_SIZE
-        if num_pages < 256 / self.fs.JOBS:
+        if num_pages < 256 / self.fs.jobs:
```

Comment:
If we start relaying on `jobs` in `remote.base` - we should adjust status optimizations to that.